Bot.Core/GhostTownCleanup.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Bot.Api;
+using Bot.Api.Database;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Bot.Core
+{
+    internal class GhostTownCleanup
+    {
+        ITownMaintenance m_startupTownTasks;
```

Comment:
All of these can probably be `private readonly`. I like doing this to prevent accidental setting of things outside of the constructor.

Reply:
Ah yeah, I usually do. Just forgot.